xml_inscriptions/transcribed/NY.NY.MMA.G.74.51.2336.xml

```diff
@@ -0,0 +1,183 @@
+                            <p>The <persName role="collector"><name key="Cesnola">Cesnola</name></persName> Collection, Purchased by subscription, <date notBefore="1873" notAfter="1877">1874–76</date>.</p>
+                        </provenance>
+                        
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Myres"/>
+                        <biblScope unit="item">1843</biblScope>
+                    </bibl>
+                    <bibl> 
+                        <ref type="external" target="https://www.metmuseum.org/art/collection/search/241890">Metropolitan Museum Web Entry</ref> 
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+        
+        <encodingDesc>
+            
+            <xi:include
+                href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for US Epigraphy controlled values</p>
+                </xi:fallback>
+            </xi:include>
+            
+        </encodingDesc>
+        
+        <profileDesc> <!-- not used --> </profileDesc>
+        
+        
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+        <!-- Record all changes here -->
+        
+        <revisionDesc>
+            <change when="2021-04-18" who="Itamar Levin">Transcription , translation, and metadata</change>
+        </revisionDesc>
+        
+    </teiHeader>
+    
+    <!--  *************************************  <facsimile>  *********************************  -->
+    <!-- Used to record images of an inscription.   -->
+    
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="NY.NY.MMA.G.74.51.2336.jpg"/>
+        </surface>
+    </facsimile>
+    
+    <text>
+        <body>
+            <div type="edition" subtype="transliteration_ltn">
+                <ab>
+                    <lb n="1" style="text-direction:l-to-r"/>e. ko. to. se. ka. te. se. ta. se. to. i.
+                    <lb n="2" style="text-direction:l-to-r"/>ti. o. i. ta. pi. te. ki. si. o. i.
+                    <lb n="3" style="text-direction:l-to-r"/>i. to. ka. i. a. za. ta. i.
+                </ab></div>
+            <div type="edition" subtype="transliteration_grk">
+                <ab>
+                    <lb n="1" style="text-direction:l-to-r"/><name key="Ἕκατος"></name> κατέστασε τῶι
+                    <lb n="2" style="text-direction:l-to-r"/>θεῶι τ᾽ <name key="Ἀμφιδεώιος" type="divine">Ἀ<choice><corr>μ</corr></choice>φιδεωίωι</name>
```

Comment:
I've changed this since I'm not convinced you're correcting what is written in the text, but simply transliterating; my understanding is that nasals before other consonants just aren't represented, so there isn't actually an error here to correct.